plugins/default/lib/openshift.py

```diff
@@ -5,18 +5,35 @@
 def oc(*args):
     return check_output(("oc",) + args)
 
-def _get_running_pod_names(oc, project):
+
+def _get_running_pods(oc, project):
     # Manually filter Running pods because of a bug in `oc get`, see:
     # https://github.com/kubernetes/kubernetes/issues/29115
-    #return oc("-n", project, "get", "pods", "--show-all=false", "-o", "name")
-    pods = json.loads(oc("-n", project, "get", "pods", "--show-all=false", "-o", "json"))
-    return [p["metadata"]["name"] for p in pods if p["status"]["phase"] == "Running"]
+    # return oc("-n", project, "get", "pods", "--show-all=false", "-o", "name")
+    pods = json.loads(oc("-n", project, "get", "pods",
+                         "--show-all=false", "-o", "json"))
+    return [p["metadata"]["name"]
+            for p in pods["items"] if p["status"]["phase"] == "Running"]
```

Comment:
it was just autopep breaking up the lines.  I'll put them back together as they were.